test/model.test.js

```diff
@@ -549,7 +544,7 @@ describe.onServer('Remote Methods', function() {
     it('Get the Change Model', function() {
       var UserChange = User.getChangeModel();
       var change = new UserChange();
-      assert(change instanceof Change);
+      assert(change instanceof app.registry.getModel('Change'));
```

Comment:
Shall we delete the declaration of [`Change`](https://github.com/strongloop/loopback/blob/5b82f94ca1a0c6db183635898ea95844b275deb0/test/model.test.js#L9) since now it's replaced with `app.registry.getModel('Change')`?